com.cuperino.qprompt.json

```diff
@@ -0,0 +1,40 @@
+{
+   "command": "qprompt",
+   "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--filesystem=host",
+      "--share=network",
+      "--device=all"
+   ],
+   "id": "com.cuperino.qprompt",
```

Comment:
id is usually the first in the manifest.

Reply:
Moved id to the start, following convention.